XamlControlsGallery/ControlPagesSampleCode/NumberBox/NumberBoxSample1_xaml.txt

```diff
@@ -0,0 +1 @@
+﻿<muxc:NumberBox Header="Enter an expression:" Value="NaN" PlaceholderText="1 + 2^2" AcceptsExpression="True" />
```

Comment:
Please move this code inline into the sample since this is a one-liner text file.